patches/ts-api-utils+0.0.39.patch

```diff
@@ -10,4 +10,4 @@ index 41ee37c..e4279d1 100644
 +	"type": "commonjs",
```

Comment:
technically we shouldn't need this patch any more if the package now correctly exports the cjs types